proactive-jupyter-kernel/kernel.py

```diff
@@ -204,6 +204,14 @@ def __trigger_pragma__(self, pragma_info):
             return self.__delete_task__
         elif pragma_info['trigger'] == 'list_submitted_jobs':
             return self.__list_submitted_jobs__
+        elif pragma_info['trigger'] == 'get_target_env_nodesources':
+            return self.__get_target_env_nodesources__  
+        elif pragma_info['trigger'] == 'get_target_env_hosts':
+            return self.__get_target_env_hosts__     
+        elif pragma_info['trigger'] == 'get_target_env_tokens':
+            return self.__get_target_env_tokens__   
+        elif pragma_info['trigger'] == 'get_target_envs':
```

Comment:
If I may suggest, environment for me it is a very wide word. It would be nice to speak more about infrastructure or resources here. I think `list_target_infrastructures` or `list_target_hdw_resources` may be more clear.